CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## [Unreleased]
 
+### Added
```

Comment:
Can you please add a newline at the end?

Reply:
Done